doc/user_guide/color_usage.rst

```diff
@@ -0,0 +1,1525 @@
+        y=alt.Y('y:N').axis(None).scale(reverse=True),
+        fill=alt.Fill('color').scale(None),
+        stroke=alt.Stroke('color').scale(None),
+        tooltip=[alt.Tooltip('color:N'), alt.Tooltip('rgb:N')]
+    )
+
+    line = alt.Chart(df_intp_routes).mark_line(
+        tooltip=True, 
+        interpolate='basis',
+        strokeWidth=3
+    ).encode(
+        x='x:N',
+        y='y:N',
+        color=alt.Color('method')
+            .scale(range=['#44aa99', '#994455', '#117733']),
+        strokeDash=alt.StrokeDash('method'),
+        order=alt.Order('index'),
+        tooltip=[
+            alt.Tooltip('color'),
+            alt.Tooltip('x'),
+            alt.Tooltip('y'), 
+            alt.Tooltip('index')
+        ]
+    )
+
+    (rect + line).configure_view(stroke=None)
+
+In the color space are the trajectories of the blue/orange color
+gradient overlayed based on the three interpolation methods
+(``rgb``, ``hcl``, ``hcl-long``).
+You can see that all interpolation methods chose a different route
+in the color space. The "``rgb``" trajectory a direct route. The
+``hcl`` trajectory is choosing a route on the bottom part of the space.
+We now also understand why there is a ``-long`` suffix
+in the interpolation method ``hcl-long``. 
+The interpolation trajectory is indeed very ``long``.
+
+The default interpolation between colors within Altair is ``hcl``. 
+Dependening on your use-case you can choose another interpolation method,
+for a complete list of interpolation options see :class:`ScaleInterpolateEnum`. 
+
+Color-blind viewers
+^^^^^^^^^^^^^^^^^^^
+   
+Around 8% of men and 0.5% of women have some form of color vision deficiency.
+To make sure your visualizations are accessible to all viewers, it's
+important to choose colors that are distinguishable even for those with 
+color vision deficiencies. By using colors that have a high degree of
+contrast, you can create visualizations that are not only informative but
+also inclusive.
+
+.. altair-plot::
+    :hide-code:    
+    :output: none
+
+    color_vision_deficiency = {
+        'redyellowblue': 9,
+        'redyellowgreen' : 9
+    }
+
+.. list-table::
+   :widths: 100
+   :header-rows: 1
+
+   * - Example
+   * - **More friendly** diverging color scheme (``redyellowblue``). 
+       This color scheme avoids using green colors, and instead uses red,
+       yellow, and blue colors.
+
+       .. altair-plot::
+         :remove-code:
+
+         plot_scheme("redyellowblue", color_vision_deficiency, cvd=True, continuous=True)
+
+   * - **Unfriendly** diverging color scale (``redyellowgreen``).
+       The red and green colors are used to represent different ends of a spectrum, 
+       but these two colors are not easy differentiated with some type of CVD.
+
+       .. altair-plot::
+         :remove-code:
+
+         plot_scheme("redyellowgreen", color_vision_deficiency, cvd=True, continuous=True)
+
+Guidelines for using colors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Love this section! One rule I would add is to start thinking about breaking our chart up into many smaller subcharts (e.g. faceting) when we get to around 5-8 different categorical colors:

![image](https://github.com/altair-viz/altair/assets/4560057/17fd1c05-1a65-449c-846a-6b7f2e08f607)